frontend/src/app/testResults/submissions/Submissions.tsx

```diff
@@ -148,7 +148,7 @@ const Submissions = () => {
                     <th scope="col">Report ID</th>
                     <th scope="col">Date/Time submitted</th>
                     <th scope="col">Records</th>
-                    <th scope="col">Status</th>
+                    <th scope="col">Submission Status</th>
```

Comment:
can I get feedback on this change? It's not exactly the same as in RS, but if we intend to eventually show Delivery Status (overallStatus) to users, it would be helpful to start differentiating between different types of status now so it's less confusing later. I propose using Submission Status and Delivery Status but am open to suggestions

Reply:
@vz3 I believe you were out for our various "what is a 'status'" conversations with ReportStream. Meredith described the issue well in the notes for this PR, but just to reiterate: ReportStream tracks a "submission" or "overall" status that simply indicates whether the submission was initially accepted or rejected at the time of upload. They also have the notion of a "delivery status", which tracks the status of the results as they are processed and make their way to the respective PHDs.

Long-term, the "delivery status" seems by far the more useful of the two measures to track and display to users. However, even ReportStream itself does not currently display this information in its UI. Therefore, for the initial release, our status displayed to the user will be the "overall" status.

The questions here and beyond are: do we want to make any effort at communicating this difference to users at this stage? How do we want to prepare for the eventual inclusion of a "delivery status" given these competing notions?

Reply:
We changed our minds 🙂 

<img width="709" alt="Screen Shot 2022-06-30 at 4 57 18 PM" src="https://user-images.githubusercontent.com/27730981/176779849-19923717-23f7-4209-80f4-9258704892e2.png">

We will just drop the `Status` column for now and re-introduce when we can show more useful delivery information.